js/stdlib/Makefile

```diff
@@ -31,7 +31,12 @@ build: package.json
 
 .PHONY: run
 run: build
-	docker-compose run stdlib
+	docker-compose run test-stdlib-ETH
+	docker-compose run test-stdlib-ALGO
+	docker-compose run test-stdlib-FAKE
+	docker-compose run test-rpc-server-ETH
+	docker-compose run test-rpc-server-ALGO
+	docker-compose run test-rpc-server-FAKE
```

Comment:
I think my preference would be to make it so we use `./reach run` for testing and then we don't have to copy the Docker set up at all. (There would be a dummy `X.rsh` to get it to run the `X.mjs` test script)